lint.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 
-set -e
+# Set err=1 if any commands exit with non-zero status as described in
+# https://stackoverflow.com/a/42219754
+err=0
+trap 'err=1' ERR
```

Comment:
Used the same approach as https://github.com/mlflow/mlflow/blob/master/dev/run-python-flavor-tests.sh